src/lib/viewers/media/_mediaBase.scss

```diff
@@ -63,6 +63,46 @@
     }
 }
 
+.bp-media-overlay-play-container {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    display: flex;
+    gap: 28px;
+    align-items: center;
+    justify-content: center;
+    color: $white;
+    font-size: 40px;
```

Comment:
We import '../../boxuiVariables' here, can we replace anything here with BUIE variables?